ts/nni_manager/training_service/reusable/environment.ts

```diff
@@ -124,7 +123,7 @@ export abstract class EnvironmentService {
 
     public abstract get hasStorageService(): boolean;
     public abstract config(key: string, value: string): Promise<void>;
-    public abstract refreshEnvironmentsStatus(environments: EnvironmentInformation[]): Promise<void>;
+    public abstract refreshEnvironmentStatus(environment: EnvironmentInformation): Promise<void>;
```

Comment:
Why change it to single one? The internal service may need batch refreshes to improve performance.

Reply:
Because now we have many environmentServices, and different environments belongs to different environmentServices. We didn't maintain a map from environmentService to environments, only have a map from environment to environmentService. environmentService does not know the environments belongs to it, we traverse environments, and get environmentService for specific environment and refresh the status. the batch refresh function can be implemented in trialDispatcher.

Reply:
TrialDispatcher should group env from same service together, and perform a batch call. Even do it every time, it won't be an overhead. But if call it for each environment, it may be an overhead. For example, on OpenPAI, it can get all jobs from one rest API call. It's more efficient than call one by one.

Reply:
upated.